src/backend/distributed/planner/extended_op_node_utils.c

```diff
@@ -77,6 +78,7 @@ BuildExtendedOpNodeProperties(MultiExtendedOp *extendedOpNode)
 		hasNonPartitionColumnDistinctAgg;
 	extendedOpNodeProperties.pullDistinctColumns = pullDistinctColumns;
 	extendedOpNodeProperties.pushDownWindowFunctions = pushDownWindowFunctions;
+	extendedOpNodeProperties.pullUpIntermediateRows = pullUpIntermediateRows;
```

Comment:
Do we still need pullDistinctColumns if we have this property? They seem to implement a similar thing.

Reply:
Created #3343 to remove `pullDistinctColumns`